frontend/src/page/MatchList.re

```diff
@@ -20,10 +20,21 @@ module GetMatchesQuery = [%graphql
 |}
 ];
 
+module GetMaxCountOfScripts = [%graphql {|
+  query($gameId: ID!) {
+    game(gameId: $gameId) {
+      maxAllowedPlayerCount
+    }
+  }
+|}];
+
 [@react.component]
 let make = (~gameId) => {
   let (version, refresh) = Utils.useRefresh();
 
+  let maxCountOfScripts =
+    Utils.useResource(GetMaxCountOfScripts.make(~gameId, ()), [|gameId, version|], data => data##game##maxAllowedPlayerCount);
+
```

Comment:
`maxAllowedPlayerCount` and `matches` should be fetched in one request, something like that:
```
  query($gameId: ID!) {
    matches(gameId: $gameId) {
       id
       name
       scriptsCount
       result {
          winner
       }
    }
    game(gameId: $gameId) {
      maxAllowedPlayerCount
    }
  }
```

Reply:
Let's check now ;)